src/object/srv_obj_migrate.c

```diff
@@ -1382,7 +1381,7 @@ migrate_fetch_update_bulk(struct migrate_one *mrone, daos_handle_t oh,
 						 min_eph,
 						 DIOF_FOR_MIGRATION | DIOF_EC_RECOV_FROM_PARITY,
 						 ds_cont);
-		if (rc > 0)
+		if (rc < 0)
```

Comment:
yeah, rc != 0 should be fine.

Reply:
> yeah, rc != 0 should be fine.

can I refine this in the later patch? since the current patch already pass the test. and all failure code are < 0 anyway.